benchmarks/nel/scripts/candidate_generation/lexical.py

```diff
@@ -0,0 +1,32 @@
+""" Candidate generation via distance in lexical space. """
+from typing import Iterable
+
+from spacy.kb import KnowledgeBase, Candidate
+from spacy.tokens import Span
+from .base import NearestNeighborCandidateSelector
+from cfuzzyset import cFuzzySet as FuzzySet
+
+
+class CandidateSelector(NearestNeighborCandidateSelector):
```

Comment:
They're both called `CandidateSelector`?

Reply:
I kinda like the _different module name_ with _same class name_ pattern if the classes have the same purpose I'm good with changing the names though. `lexical.CandidateSelector` to `lexical.LexicalNNCandidateSelector`, `embeddings.CandidateSelector` -> `embeddings.EmbeddingNNCandidateSelector`?

Reply:
I personally think it complicates discussions when talking about these things and you always have to contextualise the class names with the exact path. But yea the long names get very verbose :'(

Reply:
True. I'm good with changing the names, I think in this case the more verbose names are still ok. Should we?

Reply:
I think I'd drop the `NN` bit from the longer names, but yea otherwise they're OK to use for this project :-)